### PR TITLE
Provider API URL updated;

### PR DESCRIPTION
**Pull request by elijah-k-nebius**

### CODE REVIEW

File: slurm/versions.tf [hunk 1]
```diff
@@ -7,5 +7,5 @@ terraform {
 }
 
 provider "nebius" {
-  domain = "api.eu-north1.nebius.cloud:443"
+  domain = "api.eu.nebius.cloud:443"
```
Comment (asteny) [inline slurm/versions.tf:10]: Hello! For soperator terraform we have specific release process, we cant merge you changes to main branch. We have branch `dev/soperator`, all changes should be merged to it, then we will create and bump version of release synchroniasly with our [main soperator code base](https://github.com/nebius/soperator)
So, please make separate PR as I described

**Review by asteny — CHANGES_REQUESTED** [review]

File: slurm/versions.tf (hunk 1 above)
Comment (rdjjke) [reply slurm/versions.tf:10]: You can change the beach you merge to right in this PR on the GitHub UI

Comment (elijah-k-nebius) [reply slurm/versions.tf]: I have reverted soperator reverted changes. Please, apply them yourself however you see fit.

File: slurm/versions.tf [hunk 2]
```diff
@@ -7,5 +7,5 @@ terraform {
 }
 
 provider "nebius" {
-  domain = "api.eu.nebius.cloud:443"
+  domain = "api.eu-north1.nebius.cloud:443"
```
Comment (asteny) [inline slurm/versions.tf:10]: That change not belong to soperator

**Review by asteny — CHANGES_REQUESTED** [review]

**Review by malibora — APPROVED** [review]
